versions/2.0.0/asyncapi.md

```diff
@@ -547,12 +547,12 @@ Describes the operations available on a single channel.
 
 Field Name | Type | Description
 ---|:---:|---
-<a name="channelItemObjectRef"></a>$ref | `string` | Allows for an external definition of this channel item. The referenced structure MUST be in the format of a [Channel Item Object](#channelItemObject). If there are conflicts between the referenced definition and this Channel Item's definition, the behavior is *undefined*.
+<a name="channelItemObjectRef"></a>$ref | [Reference Object](#referenceObject) | Allows for an external definition of this channel item. The referenced structure MUST be in the format of a [Channel Item Object](#channelItemObject). If there are conflicts between the referenced definition and this Channel Item's definition, the behavior is *undefined*.
```

Comment:
If I understood correctly, this is not correct. This would result in something like:

```yaml
channels:
  test:
    $ref:
      $ref: '...'
```

I think it should be done here: https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#channelsObjectChannel. Something like `{channel} | [Channel Item Object](#channelItemObject) \| [Reference Object](#referenceObject)`